server/routes.go

```diff
@@ -70,6 +70,12 @@ func routes(h Handler, conf *config.Config) Routes {
 			Handler: h.GetFromSandbox("stderr"),
 		},
 		Route{
+			Name:    "Kill",
+			Method:  "POST",
```

Comment:
I thought of it first, but it's not actually deleting the task, it'll still be viewable through the API, so semantically DELETE felt wrong